projects/desktop/src/containers/LoginContainer.tsx

```diff
@@ -30,7 +30,14 @@ interface IForm {
  * Merges all the login form components to create
  * a useable screen
  */
-export const LoginContainer = connect()((): JSX.Element => {
+
+const mapDispatchToProps = (dispatch: Function): object => {
+    return {
+        setTheme: (theme: string): Function => dispatch({ type: 'SET_THEME', payload: { mode: theme } }),
+    }
+}

```

Comment:
What you can do here is : 
`const mapDispatchToProps = (dispatch: Function): IDispatchProps => ({
                  setTheme: (theme: string): Promise<Function> => dispatch({ type: 'SET_THEME', payload: { mode: theme } }),
})`
In the IDispatchProps there will be all the methods needed here.
In the dispatch, do something like `dispatch(setTheme(theme))`
The `setTheme` will be defined in an action file.
Tell me what you think